### ISSUE TITLE
Focus Classifier [Enhancement]

### ISSUE THREAD

**Issue by jmotschman** [yellenlab/SmartScope#1]
@calebsanfo 
We discussed adding a focus classifier value to the image acquisition stage. This would enable us to view the quality of the image focus across the chip during experiments.
This would reduce issues related to auto-focus, stage drift, camera rotation, etc.
i.e. if bad focus, we can simply re-image that time point the day-of, instead of discovering this during the analysis stage.

Relevant papers:
[Assessing microscope image focus quality with deep learning](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2087-4)
[Workflow and Metrics for Image Quality Control in Large-Scale High-Content Screens](https://journals.sagepub.com/doi/10.1177/1087057111420292) 

Eric Czech added a similar classifier previously using best z-score on the analysis end [here](https://github.com/hammerlab/celldom/blob/master/python/notebook/validation/focus_classifier.ipynb). 
High, mid, and low-focus.

**Comment by calebsanfo** [comment]
Added terminal printouts classifying each interior focus position as Good, OK, or Bad based on [Google's Microscope Image Focus Quality Classifier](https://github.com/google/microscopeimagequality) z-score.

**Comment by jmotschman** [comment]
I tested this yesterday on multiple chips and seems to be working correctly. Also, I ran through a dataset with Good/OK focus scores on the analysis code. It picks up cells, apt num, and crops correctly.

Chip: ML02
r0.8 marker_model
r0.7 single_digit_model
r0.7 cell_model

[2019-07-19-SmartScope-Cell-Test.pdf](https://github.com/yellenlab/SmartScope/files/3412609/2019-07-19-SmartScope-Cell-Test.pdf)
